packages/manager/modules/overthebox/src/overthebox/autoconfigure/dhcp/index.js

```diff
@@ -0,0 +1,9 @@
+import component from './autoconfigure-dhcp.component';
+
+const moduleName = 'ovhManagerOtbAutoconfigureDhcp';
+
+angular
+  .module(moduleName, ['ui.router'])
+  .component('autoconfigureDhcp', component);
```

Comment:
**issue(blocking):** Component name is too generic. For consistency, please keep `overTheBox` as component prefix.